code/style.css

```diff
@@ -1,4 +1,293 @@
+/* GOOGLE FONTS GOES HERE */ 
+@import url('https://fonts.googleapis.com/css2?family=Bungee&display=swap');
+
+/* GLOBAL STYLES*/ 
+* {padding: 0; margin: 0;}
+
 html {
-  background: #0025ff;
-  color: #fff;
+    background: #222222;
+    color: #E5E5E5;
+    font-family: 'Bungee'; 
+}
+
+body {
+  position:relative;
+  min-height: 100vh;
+}
+
+p {
+  font-family: 'Roboto';
+  font-size: 14px;
+}
+
+/* HEADER & NAVIGATION */
+.flex-container  {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  margin-bottom: 1rem;
+}
+
+.logo {
+    margin: 0 2em;
+}
+
+.navigation  {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+}
+
+.navigation a {
+  display: block;
+  padding: 1em;
+  color: white;
+  text-decoration: none;
+  letter-spacing: .45rem;
+}
+
+.navigation a:hover {
+  background: #00ace6;
+}
+
+/* BIG NEWS SECTION */ 
+
+.latest-news-title {
+  font-size: 40px;
+  padding: 0 0 2rem 0;
+}
+
+.big-news {
+  display: flex;
+  justify-content: center;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
+  width: auto;
+  height: auto;
+}
+
+.news {
+  display: flex;
+  flex-direction: column;
+  justify-content:center;
+  width: 1200px;
+  height: 1000px;
+
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  padding: 2rem;
+}
+
+.text-container {
+  display: flex;
+  flex-direction: column;
+  width: 1000px;
+  padding-top: 1rem;
+}
+
+.big-news-image {
+  width: 1000px;
+ }
+
+.big-news-title {
+  font-size: 32px;
+  padding-bottom: 1em;
+}
+
+.big-news-text {
+  font-size: 14px;
+  display: block;
+}
+
+.big-news-text a {
+  font-size:20px;
+  font-family:'Bungee'; 
+  text-decoration: underline;
+}
+
+.big-news-text a:hover {
+  color:#00ace6;
+}
+
+.little-title {
+  padding-top: 2rem;
+  font-size: 12px;
+  color:rgba(126, 126, 126, 0.61);
+}
+
+
+/* OTHER NEWS SECTIONS*/ 
```

Comment:
Small news is not flexed correctly. Text from news on first row is behind text on second row. The byline of the author is hidden by image on row below.  

**Potential solution**: Add padding-top on second row.

Reply:
I added padding-bottom to my .news-card class because it is the container of card and when I resize now small title shows as it should.